src/libs/merge/kdbmerge.c

```diff
@@ -46,7 +46,7 @@ static int getStatisticalValue (Key * informationKey, char * metaName)
  * @param informationKey contains the statistics in its meta information
  * @param metaName which statistic to set
  * @param value which value to set it to, must be a number
- * @returns 0 on success, 1 otherwise.
+ * @retval 0 on success, -1 otherwise.
```

Comment:
for every different retval you can make its own line.

Reply:
Is this also true for jna bindings? There it is in one line as well. If yes, we could make a good first issue out of it.

Reply:
Thank you very much for the offer! But such a small thing is not worth the effort of making a "good first issue".